tests/app/templating/summary/formatters/test_summary_item.py

```diff
@@ -0,0 +1,71 @@
+from unittest import TestCase
+
+import mock
+from mock import Mock
+
+from app.templating.summary.summary_item import SummaryItem
+
+
+class TestSummaryItem(TestCase):
+
+    def test_create_summary_item(self):
+        # Given
+        schema = mock.MagicMock()
```

Comment:
I think just calling this `schema` is confusing, `test_question_formatters.py` uses `question_schema`